demo/src/app/getting-started/getting-started.component.html

```diff
@@ -14,6 +14,16 @@
       <a href="http://v4-alpha.getbootstrap.com" target="_blank">Bootstrap CSS</a> (tested with 4.0.0-alpha.6)
     </li>
   </ul>
+
+  <h3>
+    Supported browsers
+  </h3>
+  <p>We strive to support the same browsers and versions as supported by both Bootstrap 4 and Angular, whichever is more restrictive. Check browser support notes for
+    <a href="https://angular.io/docs/ts/latest/guide/browser-support.html" target="_blank">Angular</a> and
```

Comment:
Oh, so you are going to break links :-) Yeh, we can always update when the new site goes on-line (or just wait with this PR till it happens).